README.md

```diff
@@ -176,7 +176,7 @@ interfaces more effectively by nesting your views, and pairing those views with
 
 **(1)** Follow the [setup](#get-started) instructions detailed above.
 
-**(2)** Add one or more `ui-view` to your app, give them names.
+**(2)** Add one or more `ui-view`s to your app, give them names.
```

Comment:
`**(2)** Add one or more `ui-view`s to your app with names.`  would probably be better